spec/src/modules/recommendations.js

```diff
@@ -629,6 +629,19 @@ describe('ConstructorIO - Recommendations', () => {
       });
     });
 
+    it('Should build requestURL with query params when parameters are passed', (done) => {
+      const { recommendations } = new ConstructorIO({
+        ...validOptions,
+        fetch: async (reqUrl) => ({ ok: true, json: async () => reqUrl }),
```

Comment:
Thoughts on not overriding fetch here, actually sending a request?

While this is arguably better, closer to what a unit test should be, the rest of the tests all send requests, so I'm wondering if we should stick with that.

Reply:
I think both approaches are fine but let's test the live request like Stanley mentioned. We can either 
1. Not override fetch and check both the request url and response in this test
2. Leave this test as is and create a new test that's using actual fetch and hitting the API

Reply:
added the test 👍. I think it's good to have both just to make sure the parameter is getting added to the url as expected, at least while we have this hacky workaround with parameters/network parameters